src/Spatial/Euclidean/Circle2D.cs

```diff
@@ -0,0 +1,154 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace MathNet.Spatial.Euclidean
+{
+    /// <summary>
+    /// Describes a standard 2 dimentional circle
+    /// </summary>
+    public class Circle2D : IEquatable<Circle2D>
+    {
+        /// <summary>
+        /// Creates a Circle of a given radius from a center point
+        /// </summary>
+        /// <param name="center">The location of the center</param>
+        /// <param name="radius">The radius of the circle</param>
+        public Circle2D(Point2D center, double radius)
+        {
+            Center = center;
+            Radius = radius;
+        }
+
+        /// <summary>
+        /// Creates a Circle from three points which are not colinnear
+        /// </summary>
+        /// <param name="pointA">A point on the circle</param>
+        /// <param name="pointB">A point on the circle</param>
+        /// /// <param name="pointC">A point on the circle</param>
+        public Circle2D(Point2D pointA, Point2D pointB, Point2D pointC)
```

Comment:
This is perhaps nicer as a factory method: `var circle = Cricle2D.FromThreePoints(p1, p2, p3)` Not sure if `FromThreePoints` is a good name, a bit redundant with three give three args.